docs/using_deis/process-types.rst

```diff
@@ -92,4 +92,20 @@ Use ``deis scale web=3`` to increase ``web`` processes to 3, for example. Scalin
 process type directly changes the number of :ref:`Containers <container>`
 running that process.
 
+Web vs Cmd Process Types
+------------------------
+
+When deploying to Deis using a Heroku Buildpack, Deis boots the ``web`` process type to boot the
+application server. When you deploy an application that has a Dockerfile or uses :ref:`Docker
+images <using-docker-images>`, Deis boots the ``cmd`` process type. Both are identical to
+each other in that they are exposed to the router as web applications. However, The ``cmd`` process
+type is special in which it is equivalent to running the :ref:`container` without any additional
+arguments. Every other process type is equivalent to running the relevant command that is provided
+in the Procfile.
+
+When migrating from Heroku Buildpacks to a Docker-based deployment, Deis does not explicitly
```

Comment:
All good input. I suck at articulating my thoughts, so this is all very appreciated